api/voice-react-native-sdk.api.md

```diff
@@ -139,6 +139,14 @@ namespace AuthorizationErrors {
         name: string;
         solutions: string[];
     }
+    class PayloadSizeExceededError extends TwilioError {
+        constructor(message: string);
+        causes: string[];
+        description: string;
+        explanation: string;

```

Comment:
is it necessary to have a `description` and `explanation`, can they be merged?

Reply:
Good question, it's established since before I joined the team. It's copy-pasted from really old `1.x` Voice JS SDK code. I don't think we should change it, I'm sure they had their reasons. This is also from the autogenerated errors.